std/arm64/std.porth

```diff
@@ -0,0 +1,308 @@
+/// Standard streams
```

Comment:
It would be better if Porth supported some sort of conditional compilation first so we don't have to create separate libraries for each platform

Reply:
Yeah exactly. Like C type #ifdef or something